Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini

```
@@ -6430,6 +6430,8 @@ Weapon Lazr_PatriotMissileWeapon
   ProjectileCollidesWith      = STRUCTURES
 End
 
+; Patch104p @bugfix commy2 13/09/2021 Reduce ammo against air and when assisting to match normal anti ground weapon.
+
 ;------------------------------------------------------------------------------
 Weapon Lazr_PatriotMissileWeaponAir
   PrimaryDamage               = 35.0
```

```suggestion
  PrimaryDamage               = 45.0
```
Keep damage per clip the same.

---

Dang, can't reach the assist with comments :/

---

35 * 1.333 = 46.655